### PR TITLE
Added grafana to OpenShift single step documentation.

### PR DESCRIPTION
**Pull request by sysexcontrol**
Signed-off-by: Karsten Frank <Karsten.Frank@bosch-si.com>

### CODE REVIEW

File: site/content/deployment/openshift.md [hunk 1]
```diff
@@ -146,6 +153,10 @@ In order to see the deployed components, you can use the [OpenShift Web console]
 
 In the following pictures an Eclipse Hono deployment on OpenShift is running with all the provided components.
 
+![Eclipse Hono on Openshift](../openshift_grafana.png)
+
+![Eclipse Hono on Openshift](../openshift_influxdb.png)
+
```
Comment (ppatierno) [inline site/content/deployment/openshift.md]: I would put these images at the bottom, no as first ones.

Comment (sysexcontrol) [inline site/content/deployment/openshift.md]: You are right, fully agree.
Pushed update that realizes this.

**Comment by ppatierno** [comment]
can you squash the commits then I'll rebase and merge ? thanks !

**Comment by sysexcontrol** [comment]
Squashed (but not rebased). Thank you.

**Pull request merged**